### ISSUE TITLE
[Question]: how to update properties of entities in a propertygraphstore?

### ISSUE THREAD

**Issue by justinzyw** [run-llama/llama_index#16668]
### Question Validation

- [X] I have searched both the documentation and discord for an answer.

### Question

i constructed a propertygraphstore and  there are some entities with properties and some relations. what i need to do next is to populate the entities with:
1 . some more properties which are new
2. update the contents of existing properties 

how can i achieve this please?

**Comment by justinzyw** [comment]
@dosu   does the solution above updates vector store and doc store  accordingly?

**Comment by logan-markewich** [comment]
You'll probably have the most luck just exciting the cypher queries against your graph db imo (assuming you used neo4j or falkor or nebula. Etc)

**Comment by logan-markewich** [comment]
It's easiest when your graph db is also storing embeddings. Otherwise it gets pretty complicated

**Comment by justinzyw** [comment]
yes i am using neo4j which also stores both graph and embeddings. thanks.

**Comment by justinzyw** [comment]
@dosu  could you pleas help to come up with a sample code to illustrate how to utilize upsert_nodes to fetch  info from an input document to upsert entities in an existing propertygraphstore?

**Comment by philippmaa** [comment]
@dosu
Hi everyone and thanks for this awesome library!
I am having trouble with the `Neo4jPropertyGraphStore`. I am building a `PropertyGraphIndex` using the `.from_documents` function. All documents are parsed and embeddings are created. Finally, the `.upsert_nodes` function is called and I get the error message: ```neo4j.exceptions.ConstraintError: {code: Neo.ClientError.Schema.ConstraintValidationFailed} {message: Node(0) already exists with label `__Node__` and property `id` = 5913}```. Does anybody know how to fix this? 

NOTE: the neo4j database is empty before starting.
